lib/livebook/application.ex

```diff
@@ -216,12 +217,34 @@ defmodule Livebook.Application do
             "You specified LIVEBOOK_TEAMS_NAME, but LIVEBOOK_TEAMS_KEY is missing."
           )
 
-      Livebook.Hubs.set_offline_hub(%Livebook.Hubs.Team{
+      id = "team-#{name}"
+
+      secrets =
+        if encrypted_secrets do
+          {secret_key, sign_secret} = Livebook.Teams.derive_keys(teams_key)
+
+          {:ok, json} =
+            Livebook.Teams.decrypt_secret_value(encrypted_secrets, secret_key, sign_secret)
```

Comment:
What happens if this fail? Should we pattern match and raise/log accordingly?

Reply:
Added a case clause with abort